include/field2d.hxx

```diff
@@ -149,79 +149,79 @@ class Field2D : public Field, public FieldData {
    * Direct access to the data array. Since operator() is used
    * to implement this, no checks are performed if CHECK <= 2
    */
-  inline BoutReal& operator[](const DataIterator &d) {
-    return operator()(d.x, d.y);
-  }
+  inline BoutReal &operator[](const DataIterator &d) { return operator()(d.x, d.y); }
 
   /// Const access to data array
-  inline const BoutReal& operator[](const DataIterator &d) const {
+  inline const BoutReal &operator[](const DataIterator &d) const {
     return operator()(d.x, d.y);
   }
 
   /// Indices are also used as a lightweight way to specify indexing
   /// for example DataIterator offsets (xp, xm, yp etc.) return Indices
-  inline BoutReal& operator[](const Indices &i) {
-    return operator()(i.x, i.y);
-  }
+  inline BoutReal &operator[](const Indices &i) { return operator()(i.x, i.y); }
   /// const Indices data access
-  inline const BoutReal& operator[](const Indices &i) const override {
+  inline const BoutReal &operator[](const Indices &i) const override {
     return operator()(i.x, i.y);
   }
 
   /*!
-   * Access to the underlying data array. 
-   * 
+   * Access to the underlying data array.
+   *
    * If CHECK <= 2 then no checks are performed
    *
    * If CHECK > 2 then both \p jx and \p jy are bounds checked. This will
    * significantly reduce performance.
    */
-  inline BoutReal& operator()(int jx, int jy) {
+  inline BoutReal &operator()(int jx, int jy) {
 #if CHECK > 2
-    if(!isAllocated())
+    if (!isAllocated())
       throw BoutException("Field2D: () operator on empty data");
-    
-    if((jx < 0) || (jx >= nx) || 
-       (jy < 0) || (jy >= ny) )
-      throw BoutException("Field2D: (%d, %d) index out of bounds (%d , %d)\n", 
-                          jx, jy, nx, ny);
+
+    if ((jx < 0) || (jx >= nx) || (jy < 0) || (jy >= ny))
+      throw BoutException("Field2D: (%d, %d) index out of bounds (%d , %d)\n", jx, jy, nx,
+                          ny);
```

Comment:
Can we modify the line length? Some of the changes are ok, but things like this just look a bit odd. The break could come on this line after the string